zubhub_frontend/zubhub/src/assets/js/styles/components/project/projectStyles.js

```diff
@@ -118,8 +124,9 @@ const styles = theme => ({
     marginTop: '0.5em',
     marginBottom: '0.5em',
     display: 'flex',
-    justifyContent: 'flex-start',
+    justifyContent: 'space-between',
```

Comment:
Please check this in different resolutions. It might look weird if there is too much spacing

Reply:
yeah did that before adjusting it and after adjusting..